lib/assets/javascripts/markdown-it-diaspora-linkify.js

```diff
@@ -2,69 +2,11 @@
-},{}],4:[function(require,module,exports){
 // Replace link-like texts with link nodes.
 //
 'use strict';
 
-var urlRegex = require('url-regex');
-var LINK_SCAN_RE = /www|@|\:\/\//;
+var urlRegex = /(?:^|\s)(?:(?:https?|ftp):\/\/|mailto:|xmpp:|www[^.\s]*\.)\S+/gi;
```

Comment:
The matching is generic ([JS](https://github.com/twitter/twitter-text/blob/master/js/twitter-text.js#L424), [JS](https://github.com/twitter/twitter-text/blob/master/js/twitter-text.js#L360), [Rb](https://github.com/twitter/twitter-text/blob/master/rb/lib/twitter-text/regex.rb#L374), [Rb](https://github.com/twitter/twitter-text/blob/master/rb/lib/twitter-text/regex.rb#L302)) but for some reason demands the `://` part and is then later restricted to `https?` ([JS](https://github.com/twitter/twitter-text/blob/master/js/twitter-text.js#L1368), [Rb](https://github.com/twitter/twitter-text/blob/master/rb/lib/twitter-text/validation.rb#L96)) unfortunately, so we would probably need to fork it.